docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/postpartum_depression.rst

```diff
@@ -211,9 +211,9 @@ Data Tables
       - 0
       - assumption
     * - case duration
-      - amount of time "infectedd" with PPD, in years
-      - 0.65 years (95% UI: 0.59, 0.70; truncated normal distribution). Apply uncertainty as parameter uncertainty, not individual-level heterogeneity
-      - GBD 2019 methods appendix for major depressive disorder [GBD-2019-Capstone-Appendix-PPD-mncnh]_
+      - amount of time "infected" with PPD, in years
+      - 0.65 years (95% UI: 0.59, 0.70) truncated normal distribution truncated at 0 only. Apply uncertainty as parameter uncertainty, not individual-level heterogeneity
+      - GBD 2019, 2021, and 2023 methods appendices for major depressive disorder [GBD-2019-Capstone-Appendix-PPD-mncnh]_ [GBD-2021-Capstone-Appendix-PPD-mncnh]_ [GBD-2023-Capstone-Appendix-PPD-mncnh]_
```

Comment:
Have you verified that this hasn't changed for GBD 2023? I haven't gotten a chance to look yet